test-collateral/generate-policy/src/main.rs

```diff
@@ -203,13 +210,13 @@ fn parse_command_line() -> Arguments {
                 .multiple(true),
         )
         .arg(
-            Arg::with_name("role")
-                .short("r")
-                .long("roles")
-                .value_name("ROLES")
-                .help("The set of roles of a computation participant, comma separated.")
+            Arg::with_name("capability")
```

Comment:
The relationship between the order of `capabilities` and `binary`/`certificates` is a bit confusing in pgen's arguments. Especially since it seems to require binaries be specified last on the command line.

Have you considered a different format that doesn't rely on argument order? Maybe somethings like
``` bash
pgen \
    --certificate='example-data0-cert.pem{input-0:w}' \
    --certificate='example-program-cert.pem{program.wasm:wx}' \
    --certificate='example-result-cert.pem{output:r}' \
    --binary='program.wasm{input-0:r,output:w}'
```

Reply:
Yes, the order DOES matter. I did minimum changes on top of the original version. I actually do not have good solution here. I think your solution can work, but obviously it requires a bit more parsing. I tend to argue, it might be less important for this part to be robust.